lm_eval/tasks/goat/goat_multiple_choice_sociology.yaml

```diff
@@ -0,0 +1,6 @@
+"dataset_name": "multiple_choice_sociology"
+"description": "The following are multiple choice questions (with answers) about sociology.\n\n"
+"group": "social_sciences"
+"group_alias": "social_sciences"
+"include": "_default_multiple_choice_template_yaml"
+"task": "multiple_choice_sociology"
```

Comment:
1. Let's first indicate the subject in the name, and then the type of task: `social_science_multi_choice`
2. Let's order properties in this order: dataset_name, task, description, <empty line>, group, group_alias, <empty line>, include